apps/crn-server/src/data-providers/research-outputs.data-provider.ts

```diff
@@ -69,17 +70,22 @@ export const makeDraftFilter = ({
   return query.join(' and ');
 };
 
-export interface ResearchOutputDataProvider {
-  fetchById(id: string): Promise<ResearchOutputDataObject | null>;
-  fetch(
-    options: FetchResearchOutputOptions,
-  ): Promise<ListResearchOutputDataObject>;
-  create(
-    input: ResearchOutputCreateDataObject,
-    createOptions?: { publish: boolean },
-  ): Promise<string>;
-  update(id: string, input: ResearchOutputUpdateDataObject): Promise<string>;
-}
+type ResearchOutputFetchOptions = {
+  take?: number;
+  skip?: number;
+  search?: string;
+  filter?: FetchResearchOutputFilter;
+  includeDrafts?: boolean;
+};
```

Comment:
It feels like there's going to be a lot of repetition of these properties. It might be worth insisting that all `fetch` implementations use the same options (modulo `includeDrafts`)

Reply:
`FetchOptions< FetchResearchOutputFilter> & { includeDrafts?: boolean }`